tools/msvs/vswhere_usability_wrapper.cmd

```diff
@@ -15,7 +15,7 @@ where vswhere 2> nul > nul
 if errorlevel 1 goto :no-vswhere
 set VSWHERE_REQ=-requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64
 set VSWHERE_PRP=-property installationPath
-set VSWHERE_LMT=-version "[15.0,16.0)"
+set VSWHERE_LMT=-version "[15.0,17.0)"
```

Comment:
```suggestion
set VSWHERE_LMT=-version %1
```

I suggest making this a parameter here and passing the version range explicitly from `vcbuild.bat`.

As this is now, if only VS2019 is installed and the user does not pass the `vs2019` argument, `vcbuild` fails with errors about `v141` that can be difficult to understand.